io.bit3.WhatsAppQT.yml

```diff
@@ -7,8 +7,7 @@ base-version: '5.14'
 command: WhatsAppQT
 finish-args:
 - "--share=ipc"
-- "--socket=x11"
-- "--socket=wayland"
+- "--socket=fallback-x11"
```

Comment:
Just to clarify, you still need `--socket=wayland` here, same for ThreemaQt.

Reply:
Oops, I read the [documentation](https://docs.flatpak.org/en/latest/flatpak-command-reference.html?highlight=socket#) and thought at first that the Wayland socket would then be passed through. But the second time I read this I realized that I just misunderstood it. 🙈